tsl/test/expected/remote_txn.out

```diff
@@ -30,8 +30,8 @@ CREATE OR REPLACE FUNCTION add_loopback_server(
     database               NAME = current_database(),
     port                   INTEGER = inet_server_port(),
     if_not_exists          BOOLEAN = FALSE,
-    bootstrap		   BOOLEAN = TRUE,
-    bootstrap_database     NAME = 'postgres'
+    bootstrap			   BOOLEAN = TRUE,
+	password 			   TEXT = NULL
```

Comment:
This is odd: you're removing the bootstrap database and adding a password, but I see no changes to `ts_unchecked_add_data_node` (that is `ts_data_node_add_internal`).

Ah, I see, it was not used at all so you just replaced it.

Reply:
Yes, this parameter was a left-over and should have been removed but I guess was forgotten.